scripts/install.sh

```diff
@@ -0,0 +1,21 @@
+### Source script to install k4geo 
+
+# using the nightly release
+source /cvmfs/sw-nightlies.hsf.org/key4hep/setup.sh                                                                                                                                                                                                                                                                                                              
+
+script_folder="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )" # Get the location of this script, to execute correctly other commands
+cd $script_folder/..
+
+echo "Running on xml file $xml"
+mkdir build
+cd build
+source $DD4hep_DIR/bin/thisdd4hep.sh
```

Comment:
This shouldn't be necessary, if this is sourced from the stack.
Is this for compiling against your own DD4hep?

I think this script should be dropped.

Reply:
Ah, I needed this when using my local DD4hep, so yes, it should work without